catalogue/templates/catalogue/search.html

```diff
@@ -40,7 +40,10 @@
                 <div class="tags mb-1">
                   {% for id, name in field.field.choices %}
                     {% if id|stringformat:"s" in field.value %}
-                      <span class="tag has-background-info is-rounded">{{ name }}</span>
+                      <span class="tag has-background-info is-rounded">
+                        {{ name }}
+                        <button type="button" id="{{ id }}"class="delete is-small"></button>
```

Comment:
use button `value="id_{{field.name}}_{{ forloop.counter0 }}" ` as target

Reply:
current approach does not work for TRL as checkbox values like 0, 1, 2, … are too fuzzy as a selector